client/printing/print_pb.js

```diff
@@ -404,4 +406,288 @@ proto.PrintResponse.prototype.setMessage = function(value) {
+ * @param {boolean=} opt_includeInstance Whether to include the JSPB instance
+ *     for transitional soy proto support: http://goto/soy-param-migration
+ * @return {!Object}
+ */
+proto.HealthCheckUpRequest.prototype.toObject = function(opt_includeInstance) {
+  return proto.HealthCheckUpRequest.toObject(opt_includeInstance, this);
+};
+
+
+/**
+ * Static version of the {@see toObject} method.
+ * @param {boolean|undefined} includeInstance Whether to include the JSPB
+ *     instance for transitional soy proto support:
+ *     http://goto/soy-param-migration
+ * @param {!proto.HealthCheckUpRequest} msg The msg instance to transform.
+ * @return {!Object}
+ * @suppress {unusedLocalVariables} f is only used for nested messages
+ */
+proto.HealthCheckUpRequest.toObject = function(includeInstance, msg) {
+  var f, obj = {
+    memberName: jspb.Message.getFieldWithDefault(msg, 1, "")
+  };
+
+  if (includeInstance) {
+    obj.$jspbMessageInstance = msg;
+  }
+  return obj;
+};
+}
+
+
+/**
+ * Deserializes binary data (in protobuf wire format).
+ * @param {jspb.ByteSource} bytes The bytes to deserialize.
+ * @return {!proto.HealthCheckUpRequest}
+ */
+proto.HealthCheckUpRequest.deserializeBinary = function(bytes) {
+  var reader = new jspb.BinaryReader(bytes);
+  var msg = new proto.HealthCheckUpRequest;
+  return proto.HealthCheckUpRequest.deserializeBinaryFromReader(msg, reader);
+};
+
+
+/**
+ * Deserializes binary data (in protobuf wire format) from the
+ * given reader into the given message object.
+ * @param {!proto.HealthCheckUpRequest} msg The message object to deserialize into.
+ * @param {!jspb.BinaryReader} reader The BinaryReader to use.
+ * @return {!proto.HealthCheckUpRequest}
+ */
+proto.HealthCheckUpRequest.deserializeBinaryFromReader = function(msg, reader) {
+  while (reader.nextField()) {
+    if (reader.isEndGroup()) {
+      break;
+    }
+    var field = reader.getFieldNumber();
+    switch (field) {
+    case 1:
+      var value = /** @type {string} */ (reader.readString());
+      msg.setMemberName(value);
+      break;
+    default:
+      reader.skipField();
+      break;
+    }
+  }
+  return msg;
+};
+
+
+/**
+ * Serializes the message to binary data (in protobuf wire format).
+ * @return {!Uint8Array}
+ */
+proto.HealthCheckUpRequest.prototype.serializeBinary = function() {
+  var writer = new jspb.BinaryWriter();
+  proto.HealthCheckUpRequest.serializeBinaryToWriter(this, writer);
+  return writer.getResultBuffer();
+};
+
+
+/**
+ * Serializes the given message to binary data (in protobuf wire
+ * format), writing to the given BinaryWriter.
+ * @param {!proto.HealthCheckUpRequest} message
+ * @param {!jspb.BinaryWriter} writer
+ * @suppress {unusedLocalVariables} f is only used for nested messages
+ */
+proto.HealthCheckUpRequest.serializeBinaryToWriter = function(message, writer) {
+  var f = undefined;
+  f = message.getMemberName();
+  if (f.length > 0) {
+    writer.writeString(
+      1,
+      f
+    );
```

Comment:
should this be in 4 lines?

Reply:
lmaooo this is auto generated!! we can change this